test/e2e/operator.go

```diff
@@ -179,45 +176,6 @@ var _ = Describe("ARO Operator - Geneva Logging", func() {
 	})
 })
 
-var _ = Describe("ARO Operator - Cluster Monitoring ConfigMap", func() {
-	It("must not have persistent volume set", func(ctx context.Context) {
```

Comment:
can we condition this rather than just remove it?  Condition it on the operator flag.